lib/node_modules/@stdlib/random/base/mt19937/lib/factory.js

```diff
@@ -62,19 +62,32 @@ var MATRIX_A = 0x9908b0df;
 * // returns 822569775
 */
 function factory( seed ) {
+	var seedIsArray = false;
```

Comment:
As you are not checking explicitly for a `false` value later, but a "truthy" value, this flag does not need to be explicitly set. Meaning a declaration without assignment is fine.